### PR TITLE
specify rust version

### PR DESCRIPTION
**Pull request by yanganto**

### CODE REVIEW

**Comment by yanganto** [comment]
There is some network issue in CI server, we can see in the failure CI task, and it is not related to this PR.

```
installer # error: unable to download 'https://github.com/numtide/flake-utils/archive/c0e246b9b83f637f4681389ecabcb2681b4f3af0.tar.gz': Couldn't resolve host name (6)
```

File: nix/pkgs/flake-module.nix [hunk 1]
```diff
@@ -18,6 +19,10 @@
           "format_serde_error-0.3.0" = "sha256-R4zD1dAfB8OmlfYUDsDjevMkjfIWGtwLRRYGGRvZ8F4=";
         };
       };
+      overlays = [ (import inputs.rust-overlay) ];
```
Comment (Mic92) [inline nix/pkgs/flake-module.nix:22]: drop the overlay here.

**Review by Mic92 — APPROVED** [review]

**Comment by Mic92** [comment]
bors merge

**Comment by yanganto** [comment]
bors merge

**Pull request merged**